sys/linux/socket_netlink.txt

```diff
@@ -113,16 +113,18 @@ type nlnest[TYPE, PAYLOAD] nlattr_tt[const[TYPE, int16:14], 0, 1, PAYLOAD]
 # nlattr with NLA_F_NET_BYTEORDER set (unused for now).
 type nlnetw[TYPE, PAYLOAD] nlattr_tt[const[TYPE, int16:14], 1, 0, PAYLOAD]
 
+# NL80211 already has > 320.
+define NETLINK_MAX_ATTRIBUTES	512
```

Comment:
This overshoot will lead to generation of incorrect values most of the time (even for NL80211 this is way above the max, but most other protocols have a dozen of attrs).
If there some const like NL80211_MAX_ATTR we could use here instead?

Reply:
Yes, there's such a const. I've updated the PR.

Overall it feels like an ordinary int is not a very good fit for such cases. Ideally there should be no upper bound at all, but syzkaller should just give preference to smaller values.

Reply:
Ideally there are descriptions for actual protocols :)